File: Calculator.java

import java.util.Scanner;
public class Calculator{
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);
        int choice;
        float x,y;
        System.out.print("Enter the value of x : ");
        x = input.nextFloat();
        System.out.print("Enter the value of y: ");
        y = input.nextFloat();
        System.out.print("The operations are: \n1.Addition, 2.Subtraction, 3.Multiplication, 4.Division\n");
        System.out.print("Enter the choice: ");
        choice = input.nextInt();
        if(choice == 1)
        {
            System.out.print("The Addition of "+ x +" and "+ y +" is "+(x+y));
        }
        else if(choice == 2)
        {
            System.out.print("The Subtraction of "+ x +" and "+ y +" is "+(x-y));   
        }
        else if(choice == 3)
        {
             System.out.print("The Multiplication of "+ x +" and "+ y +" is "+(x*y));
        }
        else if(choice == 4)
        {
            
             System.out.print("The Division of "+ x +" and "+ y +" is "+(x/y));
        }
        else 
        {
            System.out.println("Inavid choice");
        }
    }
}

File: Week2/Calculator.cpp
#include<iostream>
using namespace std;

int main()
{
    int choice;
    float a,b;
    cout<<"Enter the value of a: ";
    cin>>a;
    cout<<"Enter the value of b: ";
    cin>>b;
    cout<<" 1.Addition\n 2.Subtraction\n 3.Multiplication\n  4.Division\n"<<endl;
    cout<<"Enter your choice from (1-4) "<<endl;
    cin>>choice;

    switch(choice)
    {

        case 1:
        
            cout<<"The addition of "<<a<<" and "<<b<<" is "<<a+b<<endl;
            break;
        case 2:
            cout<<"The subtraction of "<<a<<" and "<<b<<" is "<<a-b<<endl;
            break;
        case 3:
            cout<<"The multiplication of "<<a<<" and "<<b<<" is "<<a*b<<endl;
            break;
        case 4:
            cout<<"The division of "<<a<<" and "<<b<<" is "<<a/b<<endl;
            break;
        default:
            cout<<"Invalid choice"<<endl;
    }
    return 0;
}
